compositorcontroller.cpp

```diff
@@ -1442,7 +1442,7 @@ namespace RdkShell
 
         if (ret)
         {
-            if ((!topmost && getNumCompositorInfo() == 0) || (topmost && focus))
+            if ((!topmost && getNumCompositorInfo() == 0) || (topmost && focus) || focus)
```

Comment:
Please change
`if ((!topmost && getNumCompositorInfo() == 0) || (topmost && focus) || focus)
            {
                gFocusedCompositor = compositorInfo;
                Logger::log(LogLevel::Information,  "rdkshell_focus create: setting focus of first application created %s", gFocusedCompositor.name.c_str());
            }`

to: 

`if ((!topmost && getNumCompositorInfo() == 0) || (topmost && focus))
            {
                gFocusedCompositor = compositorInfo;
                Logger::log(LogLevel::Information,  "rdkshell_focus create: setting focus of first application created %s", gFocusedCompositor.name.c_str());
            }
            else if (focus)
            {
                gFocusedCompositor = compositorInfo;
            }
`
This will ensure that the rdkshell_focus log line is only logged as desired.